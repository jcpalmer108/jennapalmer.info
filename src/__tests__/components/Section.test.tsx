test('test', () => {
  expect(10).toBeGreaterThan(5);
})