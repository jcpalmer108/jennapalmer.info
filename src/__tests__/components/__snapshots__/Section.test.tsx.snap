// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Section renders if only required params are passed in 1`] = `
.c0 {
  max-width: 1155px;
  margin: auto;
  border-left: solid;
  border-right: solid;
  border-color: transparent;
  border-width: 0px;
}

.c1 {
  padding: 20px;
}

@media (min-width:1155px) {
  .c0 {
    border-color: #c6c6c6;
    border-width: 1px;
  }
}

<div
  class="c0"
  data-testid="Wrapper"
>
  <div
    class="c1"
    data-testid="Children"
  >
    <p>
      Child 1
    </p>
  </div>
</div>
`;

exports[`Section renders if required and optional params are passed in 1`] = `
.c0 {
  max-width: 1155px;
  margin: auto;
  border-left: solid;
  border-right: solid;
  border-color: transparent;
  border-width: 0px;
}

.c1 {
  width: -webkit-fit-content;
  width: -moz-fit-content;
  width: fit-content;
  margin: auto;
  padding-top: 0px;
}

.c2 {
  height: 12px;
  font-size: 12;
  width: -webkit-fit-content;
  width: -moz-fit-content;
  width: fit-content;
  -webkit-column-gap: 10px;
  column-gap: 10px;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: left;
  -webkit-justify-content: left;
  -ms-flex-pack: left;
  justify-content: left;
  padding-left: 20px;
  padding-bottom: 0px;
  padding-top: 50px;
}

.c3 {
  width: 25px;
}

.c4 {
  font-size: 12px;
}

.c5 {
  padding: 0px;
}

@media (min-width:1155px) {
  .c0 {
    border-color: white;
    border-width: 0.5px;
  }
}

@media (min-width:1024px) {
  .c1 {
    margin: 0px;
    padding-top: 15px;
  }
}

@media (min-width:720px) {
  .c2 {
    height: 14px;
    font-size: 14;
  }
}

@media (min-width:1024px) {
  .c2 {
    height: 16px;
    font-size: 16;
    -webkit-transform: rotate(-90deg) translate(0px,100%);
    -ms-transform: rotate(-90deg) translate(0px,100%);
    transform: rotate(-90deg) translate(0px,100%);
    -webkit-transform-origin: bottom left;
    -ms-transform-origin: bottom left;
    transform-origin: bottom left;
    padding-bottom: calc(100% - 16px);
    padding-left: 0px;
  }
}

@media (min-width:720px) {
  .c4 {
    font-size: 14px;
  }
}

@media (min-width:1024px) {
  .c4 {
    font-size: 16px;
  }
}

<div
  class="c0"
  data-testid="Wrapper"
>
  <div
    class="c1"
    data-testid="Header"
  >
    <div
      class="c2"
      data-testid="Label"
    >
      <img
        alt="test"
        class="c3"
        src="label-line.svg"
      />
      <span
        class="c4"
      >
        TESTING
      </span>
    </div>
  </div>
  <div
    class="c5"
    data-testid="Children"
  >
    <p>
      Child 1
    </p>
  </div>
</div>
`;
